_source/logzio_collections/_metrics-sources/Prometheus-cloudwatch-exporter.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Ship Prometheus metrics from AWS CloudWatch with the OpenTelemetry Collector
+logo:
+  logofile: aws-cloudwatch.svg
+  orientation: vertical
+data-source: Amazon CloudWatch for Prometheus metrics
+templates: ["docker-metricbeat"]
+open-source:
+  - title: 
+    github-repo: 
+contributors:
+  - yotamloe
+  - yberlinger
+shipping-tags:
+  - aws
+---
+
+# logzio-aws-metrics
+
+**IMPORTANT ⚠️ :** This project is intended for specific private beta customers.
```

Comment:
Which beta customers? Prometheus beta?

Reply:
@YajB We have an include for this. Try  {% include page-info/early-access.md type="beta" %}